trio/_ssl.py

```diff
@@ -897,16 +914,16 @@ class SSLListener(Listener[SSLStream], metaclass=Final):
 
     def __init__(
         self,
-        transport_listener,
-        ssl_context,
+        transport_listener: Listener[Stream],
+        ssl_context: _stdlib_ssl.SSLContext,
         *,
-        https_compatible=False,
-    ):
+        https_compatible: bool = False,
+    ) -> None:
```

Comment:
FWIW this `-> None` is probably also unnecessary.

Reply:
Yeah, I haven't seen any reason why it would change anything.